e2e-pw/src/oss/specs/smoke-tests/embeddings.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { test as base } from "src/oss/fixtures";
+import { GridPom } from "src/oss/poms/grid";
+import { PanelPom } from "src/oss/poms/panels/panel";
+import { EmbeddingsPom } from "src/oss/poms/panels/embeddings-panel";
+import { getUniqueDatasetNameWithPrefix } from "src/oss/utils";
+
+const datasetName = getUniqueDatasetNameWithPrefix("smoke-quickstart");
+
+const test = base.extend<{ grid: GridPom; panel: PanelPom }>({
+  grid: async ({ page, eventUtils }, use) => {
+    await use(new GridPom(page, eventUtils));
+  },
+  panel: async ({ page }, use) => {
+    await use(new PanelPom(page));
+  },
+  embeddings: async ({ page, eventUtils }, use) => {
+    await use(new EmbeddingsPom(page, eventUtils));
+  },
+});
+
+test.beforeAll(async ({ fiftyoneLoader }) => {
+  await fiftyoneLoader.executePythonCode(
+    `
+        import fiftyone as fo
+        import fiftyone.zoo as foz
+        import fiftyone.brain as fob
+        import numpy as np
+
+        dataset = foz.load_zoo_dataset("quickstart", max_samples=5, dataset_name="${datasetName}")
+        dataset.persistent = True
+
+        embeddings = np.random.random((5, 512))
```

Comment:
can we seed numpy PRNG so that it's always the same embeddings?

Reply:
absolutely!